src/picklescan/scanner.py

```diff
@@ -114,15 +114,13 @@ def __str__(self) -> str:
     "subprocess": "*",
     "sys": "*",
     "operator": "attrgetter",  # Ex of code execution: operator.attrgetter("system")(__import__("os"))("echo pwned")
+    "pickle": "*",
+    "_pickle": "*",
```

Comment:
I would've sworn it was double `_`, did your tests show only a single underscore?

Reply:
Yes -- here is the disassembly of the version 0 and version 4 test cases:

```bash
> python -m pickletools malicious13a.pkl
    0: c    GLOBAL     'pickle loads'
   14: p    PUT        0
   17: (    MARK
   18: c        GLOBAL     '_codecs encode'
   34: p        PUT        1
   37: (        MARK
   38: V            UNICODE    'I12345\n.'
   53: p            PUT        2
   56: V            UNICODE    'latin1'
   64: p            PUT        3
   67: t            TUPLE      (MARK at 37)
   68: p        PUT        4
   71: R        REDUCE
   72: p        PUT        5
   75: t        TUPLE      (MARK at 17)
   76: p    PUT        6
   79: R    REDUCE
   80: p    PUT        7
   83: .    STOP
highest protocol among opcodes = 0
```
```bash
> python -m pickletools malicious13b.pkl
    0: \x80 PROTO      4
    2: \x95 FRAME      36
   11: \x8c SHORT_BINUNICODE '_pickle'
   20: \x94 MEMOIZE    (as 0)
   21: \x8c SHORT_BINUNICODE 'loads'
   28: \x94 MEMOIZE    (as 1)
   29: \x93 STACK_GLOBAL
   30: \x94 MEMOIZE    (as 2)
   31: C    SHORT_BINBYTES b'I12345\n.'
   41: \x94 MEMOIZE    (as 3)
   42: \x85 TUPLE1
   43: \x94 MEMOIZE    (as 4)
   44: R    REDUCE
   45: \x94 MEMOIZE    (as 5)
   46: .    STOP
highest protocol among opcodes = 4
```